pkg/logcli/output/output_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func TestNewLogOutput(t *testing.T) {
-	options := &LogOutputOptions{time.UTC, false, false}
+	options := &LogOutputOptions{time.UTC, false, false, ""}
```

Comment:
could you add some tests around the additional cases you've introduced?

Reply:
Does adding some of the allowed values enough or any pointer on what should I test?

https://github.com/grafana/loki/blob/1cce6680793e4fe87eaf5c9eb80fc55f928e18a9/cmd/logcli/main.go#L36

Reply:
Added 2 tests within the following as additional permutations: https://github.com/grafana/loki/blob/e8724f7d909a23e110e2b62c3b20d56fb057418c/pkg/logcli/output/default_test.go#L14

To note, all other `LogOutputOptions{}` invocations does it with named argument and this is the only instance it is constructed not as named argument (sorry my terminology) so I assume it should not be used as such and changed the test construction to follow the rest of the codebase.

Also at the place it is being used, there is fallback to the default formatter to preserve compatibility

https://github.com/grafana/loki/blob/e8724f7d909a23e110e2b62c3b20d56fb057418c/pkg/logcli/output/default.go#L22-L25